Makefile

```diff
@@ -42,6 +42,33 @@ endif
 build:
 	$(GO) build $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS) -o $(PKGNAME) $(BUILDFILES)
 
+PLATFORMS := darwin linux windows
+
+WIN_FLAGS = -ldflags "-X main.Version=$(VERSION) -linkmode=internal -s -w -extldflags '-static' -extld=x86_64-w64-mingw32-gcc"
+
+buildNatives:
+	@for platform in $(PLATFORMS); \
+	do \
+		arch="amd64"; \
+		output_name="$(PKGNAME)-$$platform"; \
+		if [ $$platform = windows ]; then \
+			echo $$output_name; \
+			echo "asdasddassad"; \
+			wintags='$(TAGS) nolzma'; \
+			env CGO_ENABLED=1 CC=x86_64-w64-mingw32-gcc CXX=x86_64-w64-mingw32-g++ GOOS=$$platform GOARCH=$$arch $(GO) build $(WIN_FLAGS) -tags $$wintags $(BUILDV) $(BUILDFILES) ; \
+		else \
+			echo $$output_name; \
+			env CGO_ENABLED=0 GOOS=$$platform GOARCH=$$arch $(GO) build -a $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS) -o $$output_name $(BUILDFILES) ; \
```

Comment:
Why `CGO_ENABLED=0`? Lzma requires it, no?

Should we just lose `CGO_ENABLED` from both of these lines? It's already set globally at the top.

Reply:
without the CGO_ENABLED=0 the darwin build will crash

as for the lzma requirement: my guess was that if it runs in the container, it should be able to be run on linux... is there a specific functionality that I should try to run in order to see if it works with lzma?

Reply:
In the tests?
Locally you can just run mender artifact, and then try to create an artifact with the zlma (xz-extension) option for the artifact.

Reply:
> as for the lzma requirement: my guess was that if it runs in the container, it should be able to be run on linux... is there a specific functionality that I should try to run in order to see if it works with lzma?

Yes, try this:

```
./mender-artifact --compression lzma write rootfs-image -n myname -t mydevice -f /dev/null
```

It's ok if this doesn't work on Mac, but it should work on Linux, and I suspect that `CGO_ENABLED=0` will prevent that.